src/Stratis.Bitcoin.Cli/Program.cs

```diff
@@ -8,72 +9,144 @@
 using Newtonsoft.Json;
 using Stratis.Bitcoin.Configuration;
 using Stratis.Bitcoin.Features.RPC;
+using Stratis.Bitcoin.Utilities.Extensions;
 
 namespace Stratis.Bitcoin.Cli
 {
     class Program
     {
         /// <summary>
         /// The expected sequence of arguments:
-        ///    1) [network-name] [rpc-command] [rpc-params].
+        ///    1) [network-name] [options] [rpc-command] [rpc-params].
         /// OR
-        ///    2) [network-name] [api-controller "/" api-command] [api-params].
+        ///    2) [network-name] [options] [api-controller "/" api-command] [api-params].
         /// </summary>
         public static void Main(string[] args)
         {
             try
             {
+                // Preprocess the command line arguments
+                List<string> argList = new List<string>(args);
+
+                string networkName = null;
+                if (argList.Count > 0)
```

Comment:
```.Any()``` is better

Reply:
Is there any difference between the two?